server/global/importationUtils.js

```diff
@@ -17,6 +15,16 @@ export const isString = (v) => typeof v === "string" || v instanceof String;
  */
 export const isNumber = (v) => typeof v === "number" || v instanceof Number;
 
+/**
+ * Checks if two strings are equal regardless of case and accents
+ * @param {string} str1 The first string
+ * @param {string} str2 The second string
+ * @returns {boolean}
+ */
+export function isSameString(str1, str2) {
+  return Boolean(String(str1).localeCompare(String(str2), "fr", { sensitivity: "base" }) === 0);
```

Comment:
Nitpick: I think the cast of `===` to `Boolean` is useless
```suggestion
  return 0 === String(str1).localeCompare(String(str2), "fr", { sensitivity: "base" });
```